python-main.js

```diff
@@ -1498,6 +1505,13 @@ function web_editor(config) {
         // Firmware at the end of the HTML file has to be loaded first
         setupFilesystem();
     });
+
+    // Catch any unhandled events
+    window.addEventListener("unhandledrejection", function(promiseRejectionEvent) { 
```

Comment:
What's the main reason we need to do this? Will this hide any errors we haven't foreseen during development?

Reply:
I used it to help catch and diagnose any errors that aren't handled

I left it in to keep the console clean, but thinking about it hiding everything is probably a bad idea

Reply:
Removed in bccbb9a3a081baccbb5dfa47c57aaa0f7259d299.